modules/books.js

```diff
@@ -4,14 +4,14 @@ const mongoose = require('mongoose');
 const { Schema } = mongoose;
 
 const bookSchema = new Schema({
-  Books:Array,
+  Books: Array,
 })
 
 const booksModel = mongoose.model('books', bookSchema);
 
 let seedFunction = () => {
```

Comment:
- Feedback here: Consider making this an async function then use`create()` to create a new book based off seed data using title, description, status, email